assets/svgs/getting-started-24.svg

```diff
@@ -0,0 +1,10 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+    <rect x="0.5" y="12.5" width="10" height="10" rx="2" ry="2" style="fill:none;stroke:#000;stroke-linecap:round;stroke-linejoin:round"/>
```

Comment:
Please replace `#000` with `currentColor` so the icon color is inherited from the text color.